### ISSUE TITLE
PHP 8 error while using latest docker build

### ISSUE THREAD

**Issue by NjayNL** [jgeusebroek/docker-spotweb#46]
Hi,

I upgraded to the the latest docker image which worked fine in the past but now i get this error message on the main page:

Deprecated: Function libxml_disable_entity_loader() is deprecated in /var/www/spotweb/lib/Bootstrap.php on line 82

when clicking mark everything as read:

Deprecated: Function libxml_disable_entity_loader() is deprecated in /var/www/spotweb/lib/Bootstrap.php on line 82
{"result":"success","data":[],"info":[],"warnings":[],"errors":[]}

can you look into the code?

**Comment by jgeusebroek** [comment]
What version of the image are you using?

**Comment by NjayNL** [comment]
i used the latest tag i reverted back to the php7.4 tag and the erorr is gone only the mark everthing as read button is giving the following blank page with this error code: 

{"result":"success","data":[],"info":[],"warnings":[],"errors":[]}

**Comment by jgeusebroek** [comment]
The latest tag has not been PHP version 8, are you sure you used my image? Only latest tag is support.

**Comment by NjayNL** [comment]
Yes i am using the latest tag just tested again and the error is back on the main page

Deprecated: Function libxml_disable_entity_loader() is deprecated in /var/www/spotweb/lib/Bootstrap.php on line 82

using docker on synology dsm 7

**Comment by jgeusebroek** [comment]
That's very strange.

What does this output:

`docker image ls jgeusebroek/spotweb`.
`docker exec -ti spotweb php -v`

**Comment by NjayNL** [comment]
root@DS916:~# docker image ls jgeusebroek/spotweb.
REPOSITORY   TAG       IMAGE ID   CREATED   SIZE

root@DS916:~# docker exec -ti spotweb php -v
PHP 8.0.13 (cli) (built: Nov 22 2021 09:50:43) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.13, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.13, Copyright (c), by Zend Technologies

**Comment by NjayNL** [comment]
root@DS916:~# docker image ls jgeusebroek/spotweb
REPOSITORY            TAG       IMAGE ID       CREATED        SIZE
jgeusebroek/spotweb   latest    6a0b7fcb3a6f   3 months ago   386MB
root@DS916:~#

**Comment by jgeusebroek** [comment]
That is indeed the latest version. I'm stumped. I cannot replicate your problem, I don't see any errors.

**Comment by NjayNL** [comment]
I rolled back to a tag from a year ago and the errors are gone so for my config it’s something in the latest image 😊 I will start fresh hopefully that will work thanks for your support

Van: Jeroen Geusebroek ***@***.***>
Verzonden: Friday, 8 April 2022 09:14
Aan: jgeusebroek/docker-spotweb ***@***.***>
CC: NjayNL ***@***.***>; Author ***@***.***>
Onderwerp: Re: [jgeusebroek/docker-spotweb] PHP 8 error while using latest docker build (Issue #46)


That is indeed the latest version. I'm stumped. I cannot replicate your problem, I don't see any errors.

**Comment by NjayNL** [comment]
Fixed ! removed all images and rebuild a new one restored my config and done
Only thing remaining is the upgrade-db wont work with the new mariadb 😊

Van: Nick Jamont
Verzonden: Friday, 8 April 2022 09:16
Aan: jgeusebroek/docker-spotweb ***@***.***>; jgeusebroek/docker-spotweb ***@***.***>
CC: Author ***@***.***>
Onderwerp: RE: [jgeusebroek/docker-spotweb] PHP 8 error while using latest docker build (Issue #46)

I rolled back to a tag from a year ago and the errors are gone so for my config it’s something in the latest image 😊 I will start fresh hopefully that will work thanks for your support

Van: Jeroen Geusebroek ***@***.******@***.***>>
Verzonden: Friday, 8 April 2022 09:14
Aan: jgeusebroek/docker-spotweb ***@***.******@***.***>>
CC: NjayNL ***@***.******@***.***>>; Author ***@***.******@***.***>>
Onderwerp: Re: [jgeusebroek/docker-spotweb] PHP 8 error while using latest docker build (Issue #46)


That is indeed the latest version. I'm stumped. I cannot replicate your problem, I don't see any errors.

**Comment by NjayNL** [comment]
will test further tomorrow thanks for the quick help
________________________________
